index.html

```diff
@@ -205,6 +205,19 @@ <h3>Skills</h3>
           <hr>
         </div>
       </section>
+      <section class="Contact-Me">
+        <div class="background-2"> 
+        <h2 class="contact-title">Contact Me</h2>
+        <p class="contact-description">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form class="Conact-Form" method="post" action="https://formspree.io/f/mbjwkepg">
+        <input type="text" name="FullName" placeholder="Enter your name" maxlength="30" required>
+         <input type="email" name="email" placeholder="Enter your email" required>
+         <textarea name="message" placeholder="Enter message" maxlength="500" rows="10" required></textarea>
+         <button class="viewpro-btn float-left" type="submit" name="submit-btn">Get in touch</button>
+        </form>
+     </div>
+        <div class="end-line"></div>
+      </section>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![sssa](https://user-images.githubusercontent.com/69638013/159517508-dd565e98-d510-46d1-9c1a-c4e3ad75e7b9.png)  | ![sty](https://user-images.githubusercontent.com/69638013/159517457-313adcad-4b4a-445e-ad82-85b319ece628.png)

- [ ] Good job adding all the functionalities,  Kindly change the color of the message text placeholder so your design looks exactly like the Figma design

- [ ] Kindly change the appearance of your button to white it currently has a default background color of `#f7f7f9`

- [ ] Kindly remove the div with the endline as this is not required to be in your project 👍🏿